Flick/Controllers/FriendsViewController.swift

```diff
@@ -30,8 +29,17 @@ class FriendsViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        title = "Friends"
         view.backgroundColor = .offWhite
-        setupNavigationBar()
+
+        headerView.backgroundColor = .movieWhite
+        headerView.clipsToBounds = false
+        headerView.layer.masksToBounds = false
```

Comment:
if all these header views are the same, we can probably just make a component for it